save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entities/LnkExecutionTestSuite.kt

```diff
@@ -0,0 +1,26 @@
+package com.saveourtool.save.entities
+
+import com.saveourtool.save.spring.entity.BaseEntityWithDto
+import javax.persistence.Entity
+import javax.persistence.JoinColumn
+import javax.persistence.ManyToOne
+
+/**
+ * @property execution
```

Comment:
What's the point of this dummy KDoc comment?

To pacify `diktat` and make it happy?
But the corresponding `diktat` rule is enabled on purpose, I believe.

You should either disable the rule globally, or suppress it locally, or go ahead and add the proper KDoc comment.

Reply:
Ok. Fixed KDoc comment